plugins/connection/aws_ssm.py

```diff
@@ -497,7 +497,11 @@ def _flush_stderr(self, subprocess):
 
     def _get_url(self, client_method, bucket_name, out_path, http_method):
         ''' Generate URL for get_object / put_object '''
-        client = self._get_boto_client('s3')
+        if self.get_option('region') is None:
+            region_name = 'us-east-1'
+        else:
+            region_name = self.get_option('region')
```

Comment:
You can actually just do, and this will also cover the case where the `region` is an empty string:

```python
region_name = self.get_option('region') or 'us-east-1'
```

Reply:
This has been addressed with the new commits.